plugin.xml

```diff
@@ -57,7 +45,6 @@
 
         <framework src="src/android/build.gradle" custom="true" type="gradleReference" />
 
-        <source-file src="src/android/org/kisio/CDVNavitiaSDK/CDVNavitiaSDK.java" target-dir="src/org/kisio/CDVNavitiaSDK" />
         <source-file src="src/android/org/kisio/CDVNavitiaSDKUX/CDVNavitiaSDKUX.java" target-dir="src/org/kisio/CDVNavitiaSDKUX" />
     </platform>
-</plugin>
+</plugin>
```

Comment:
Please don't remove empty line at the end of the file. This will potentially cause problems. 
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file